com.github.VescTool.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: com.github.VescTool
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: vesc_tool
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+modules:
+  - name: vesc-tool
+    buildsystem: qmake
+    sources:
+      - type: git
+        url: https://github.com/vedderb/vesc_tool
+        commit: b0632c42e896175f9d731be5f146f0202a0fe17c
+      - type: patch
+        path: install_linux.patch
+  - name: metainfo
+    buildsystem: simple
+    build-commands:
+      - install -D -m0644 com.github.VescTool.metainfo.xml /app/share/metainfo/com.github.VescTool.metainfo.xml
+    sources:
+      - type: file
+        path: com.github.VescTool.metainfo.xml
+  - name: desktop
+    buildsystem: simple
+    build-commands:
+      - install -D -m0644 com.github.VescTool.png /app/share/app-info/icons/flatpak/128x128/com.github.VescTool.png
```

Comment:
Usually the desktop and metainfo are put as source in the main module. No need for 2 extra modules.

Reply:
ok